pymtl3/passes/backends/sverilog/import_/test/ImportedObject_test.py

```diff
@@ -55,6 +56,40 @@ def construct( s ):
   a._tv_out = tv_out
   do_test( a )
 
+def test_reg_external_trace( do_test ):
+  class VRegTrace( Component ):
+    def construct( s ):
+      s.in_ = InPort( Bits32 )
+      s.out = OutPort( Bits32 )
+      s.config_sverilog_import = ImportConfigs(
+          vl_src = get_dir(__file__)+'VRegTrace.sv',
+          vl_include = [ get_dir(__file__) ],
+          port_map = {
+            "in_" : "d",
+            "out" : "q",
+          },
+          external_trace = True,
+      )
+  a = VRegTrace()
+  a.elaborate()
+  ipass = ImportPass()
+  a.config_sverilog_import.fill_missing( a )
+  a = ipass.get_imported_object( a )
+  a.apply( SimulationPass() )
+
+  a.in_ = Bits32(1)
+  a.tick()
+  assert a.line_trace().replace(' ', '') == 'q=0'
```

Comment:
is it possible to check equality with space?

Reply:
Yes